lib/chat_models/ollama_ai_fields.ex

```diff
@@ -0,0 +1,78 @@
+defmodule LangChain.ChatModels.OllamaAIFields do
+  defmacro fields do
```

Comment:
Why do this in a macro and a separate file?

Reply:
With so many fields, the chat_ollama_ai.ex was getting hard to read. I can inline the fields back in though if that is your preference. If you have any alternatives for DRY'ing those fields up let me know

Reply:
Thank, I see you already did. :+1: